src/main/kotlin/lotto/domain/LottoNumber.kt

```diff
@@ -10,5 +11,8 @@ class LottoNumber(
 
     companion object {
         val LOTTO_NUMBERS: List<Int> = (1..45).toList()
```

Comment:
```suggestion
        val LOTTO_NUMBERS: List<Int> = (1..45).map { LottoNumber(it) }
```
로또 번호 객체를 캐싱하는건 어떠실까요 ?